### DISCUSSION TITLE
Changing title font type

### DISCUSSION THREAD

**Discussion by DriptD** [mtex-toolbox/mtex#685] [Ask Anything]
Hello everyone,

I was wondering if it's possible to change the font type of the title (please see the attached image) to that of the other annotations in a pole figure plot. Does anyone know how to do it?

Also, is it possible to get a "Super Title" in a mtex figure window with several subplots?

I'm using MTEX v5.4.0 in MATLAB 2020b.

Thanks !!

![Pole Figure](https://user-images.githubusercontent.com/77139598/103978964-b0eff600-51a2-11eb-8fcf-63a67fe33812.jpg)

Regards,
Dripta

**Comment by ralfHielscher** [comment]
You may use the function `mtexTitle` to add a title to an arbitrary axis with arbitrary options passed, e.g., 'fontName' check out matlab text properties to find a full list. The function mtexTitle has also an option `global` which allows to draw a global title for all subfigures.

see https://github.com/mtex-toolbox/mtex/blob/develop/compatibility/mtexTitle.m
unfortunately this file lacks documentation :(

Ralf.

**Reply by DriptD** [reply]
Dear Dr. Hielscher,

Thanks for the response, I did get the global title. However, I'm still not able to change the font type. I executed the following command but did not get the result (attached) I was expecting. Not sure where am I going wrong !!

mtexTitle('TITLE','fontName','Arial','FontSize',24,'FontWeight','bold','global')
![FontName](https://user-images.githubusercontent.com/77139598/104087281-9aba6680-5284-11eb-87cf-16c74d030ea6.jpg)

sgtitle() works partially. But then the title overlaps the top of the pole figure plots on the 1st row. Is it possible to set the position of the 1st row of the pole figures so that they can be shifted a little towards the bottom of the window?

Thanks !!

Dripta
